### DISCUSSION TITLE
Linux distribution

### DISCUSSION THREAD

**Discussion by FormationOLB** [TOSIT-IO/TDP#29] [Q&A] [answered]
Hello all,
Is there a linux distribution recommended for TDP ? Compilation and deployment both.
I tried only on CentOS Linux release 7.9.2009. 
Is someone test on Alma Linux or Ubuntu ? 

Thank you for your help.

**Comment by rpignolet** [comment, marked as answer]
At DGFiP, we use Centos 7 for TDP deployment, other distributions are not tested.

For compilation we use container : https://github.com/TOSIT-IO/TDP/tree/main/build-env based on Ubuntu.

**Reply by FormationOLB** [reply]
Ok, thank you. I will try on AlmaLinux, just to fun.

**Reply by BenoitBaril** [reply]
Hello @FormationOLB,
Did you try AlmaLinux ?
If yes, what is the result on your side please ?

**Reply by jsanc525** [reply]
Good morning,
I've managed to get a vagrant cluster of TDP getting started running with both Rocky Linux 8 and Alma Linux 8. 
The master branch works well with centos7 as is. Master branch of getting-started will point to tagged version of the collections 0.1.
The dev branch should work with rocky8 (and should work with alma-will test soon!) as it points to the latest commits of the collections

**Comment by martin-orange-dds** [comment]
Hello,
We started a POC in my company with Ubuntu20.04 few weeks ago.
Most features are working. Other aren't. We managed to run some jobs.

Not sure if we still get some budget for TDP in the upcoming weeks.
We will try to PR the adaptation we needed regardless.

**Reply by mehdibn** [reply]
@martin-orange-dds : good news, we can plan a point together in order to discuss this integration ?

**Comment by morihuang** [comment]
Hi all!
Currently on AlmaLinux8 with py3.9.18. Still in the very beginning stage.
I've tried AlmaLinux8 with py3.10.12 with several changes in the ansible playbook and has luckily been able to implement in the end.
